### PR TITLE
Some small fixes to ease versioning

### PR DESCRIPTION
**Pull request by mmattel**
Some small fixes to ease versioning

### CODE REVIEW

**Review by kobergj — APPROVED** [review]

File: site.yml [hunk 1]
```diff
@@ -42,6 +42,10 @@ asciidoc:
 #   ocis
     latest-ocis-version: 'next@'   # do not change, soft set, correctly defined via antora.yml
     previous-ocis-version: 'next@' # do not change, soft set, correctly defined via antora.yml
+    # these versions are just for printing like in releases but not used for referencing
+    # needs to be changed here and not in the docs-ocis repo to be properly shown on the web
+    # the following versions get printed on <all> build versions where applicapable
```
Comment (phil-davis) [inline site.yml:47]:
```suggestion
    # the following versions get printed on <all> build versions where applicable
```

File: site.yml [hunk 2]
```diff
@@ -42,6 +42,10 @@ asciidoc:
 #   ocis
     latest-ocis-version: 'next@'   # do not change, soft set, correctly defined via antora.yml
     previous-ocis-version: 'next@' # do not change, soft set, correctly defined via antora.yml
+    # these versions are just for printing like in releases but not used for referencing
+    # needs to be changed here and not in the docs-ocis repo to be properly shown on the web
```
Comment (phil-davis) [inline site.yml:46]: "not in the docs-ocis repo"
but this is the docs-ocis repo.
I don't understand what this means.

Comment (mmattel) [inline site.yml:46]:
```suggestion
    # needs to be changed here and mandatory in the docs repo to be properly shown on the web
```
copy paste error 🤦‍♂️

Comment (mmattel) [reply site.yml]: Thanks for catching, fixed

**Pull request merged**